app/assets/javascripts/discourse/admin/components/admin-filter-controls.gjs

```diff
@@ -1,13 +1,16 @@
 import Component from "@glimmer/component";
```

Comment:
All the changes in this file are to allow for multiple dropdowns in this control, which I needed on this page

<img width="1172" height="144" alt="image" src="https://github.com/user-attachments/assets/7eb4d59f-f83e-4500-94cf-abb0a663a449" />